src/dcfldd.c

```diff
@@ -202,6 +203,7 @@ Enhanced version of dd for forensics and security.\n\
   vf=FILE                  verify that FILE matches the specified input\n\
   verifylog=FILE           send verify results to FILE instead of stderr\n\
   verifylog:=COMMAND       exec and write verify results to process COMMAND\n\
+  diffwr=[on|off]          write only to output if destination block content differs\n\
```

Comment:
This sentence is mostly ok, but `only write to output if...`  would convey a clearer meaning.